src/lib/db/event-store.ts

```diff
@@ -412,7 +412,9 @@ class EventStore implements IEventStore {
             .update({ announced: true })
             .where('announced', false)
             .whereNotNull('announced')
-            .returning(EVENT_COLUMNS);
+            .where('created_at', '>', new Date(Date.now() - 1000 * 60 * 60))
```

Comment:
```suggestion
            .where('created_at', '>', new Date(Date.now() - 24 * 1000 * 60 * 60))
```

Reply:
I'd use: `import { addDays, subDays } from 'date-fns';`

Reply:
Ah, yeah, that would work as well! Which ever you think is better 💁🏼  Probably safer to go for 24.

Reply:
Thanks! I added subDays but I'm not 100% confident in my querying skills so I'll write a test